gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/io/graphml/GraphMLReader.java

```diff
@@ -381,7 +383,18 @@ public Builder batchSize(final long batchSize) {
             return this;
         }
 
+        /**
+         * the key to use as the inputFactory when a caller wants to pass XMLInputFactory with its own configuration.
+         */
+        public Builder xmlInputFactory(final XMLInputFactory inputFactory) {
+            this.inputFactory = inputFactory;
+            return this;
+        }
+
         public GraphMLReader create() {
+            if (this.xmlInputFactory() == null) {
```

Comment:
does this line compile? `xmlInputFactory` isn't a method is it? shouldn't you remove those parenthesis and just do: `if (this.xmlInputFactory == null) {`

Reply:
Sorry I was lazy... this should have been
```
if (this.inputFactory == null)
```
I updated the PR. Thanks!